src/stream_base-inl.h

```diff
@@ -72,6 +72,10 @@ void StreamBase::GetFD(Local<String> key,
                        const PropertyCallbackInfo<Value>& args) {
   StreamBase* wrap = Unwrap<Base>(args.Holder());
 
+  // Mimic implementation of StreamBase::GetFD() and UDPWrap::GetFD().
+  if (wrap == nullptr)
+    return args.GetReturnValue().Set(-1);
```

Comment:
Because if the value isn't a two byte aligned Smi then the application aborts with:
```
FATAL ERROR: v8::Object::GetAlignedPointerFromInternalField() Not a Smi
```